### PR TITLE
[TSOPS-1304] Elasticasearch index version check

### PR DESCRIPTION
**Pull request by patrickhargett**
See TSOPS-1304

### CODE REVIEW

File: support_bundle/get_support_bundle.sh [hunk 1]
```diff
@@ -298,27 +298,30 @@ main() {
...
 
             echo "Fetching ElasticSearch SSL Certificate Expiration Dates"
             kubectl ${KUBE_OPTS} exec -it ${pod}  -c elasticsearch -- openssl x509 -in /usr/share/elasticsearch/config/node.pem -noout -enddate | tee -a ${LOG_DIR}/elasticsearch/${pod}/elasticsearch_node_pem_expiration.log
             kubectl ${KUBE_OPTS} exec -it ${pod}  -c elasticsearch -- openssl x509 -in /usr/share/elasticsearch/config/admin.pem -noout -enddate | tee -a ${LOG_DIR}/elasticsearch/${pod}/elasticsearch_admin_pem_expiration.log
             kubectl ${KUBE_OPTS} exec -it ${pod}  -c elasticsearch -- openssl x509 -in /usr/share/elasticsearch/config/root-ca.pem -noout -enddate | tee -a ${LOG_DIR}/elasticsearch/${pod}/elasticsearch_root_ca_pem_expiration.log
 
+            echo "Fetching Elasticsearch Indexes with version 5"
+            kubectl ${KUBE_OPTS} exec -it ${pod} -- bash -c "${ELASTIC_CURL}/_all/_settings/index.version\*?pretty" | tee -a ${LOG_DIR}/elasticsearch/${pod}/elasticsearch_index_version_5.log
```
Comment (dthomson-sysdig) [inline support_bundle/get_support_bundle.sh:323]: is this log file generic to all the index versions that we will parse later?  Just wondering if the output filename should be more generic.

Comment (mbreitung) [reply support_bundle/get_support_bundle.sh:323]: is this missing `-c elasticsearch `?

**Comment by patrickhargett** [comment]
This is a duplicate. Closing.

**Comment by mbreitung** [comment]
closing in favor of #210

**Pull request closed without merging**